_dashboards/run-queries.md

```diff
@@ -0,0 +1,115 @@
+---
+layout: default
+title: Running queries in the console
+nav_order: 110
+---
+
+# Running queries in the console
+
+You can use the OpenSearch DevTools Console to send queries to OpenSearch. 
```

Comment:
I think @natebower should have some input on this. This should be "Dev Tools" (with a space), but more broadly, what should we refer to this as? Elasticsearch docs say "Interact with the REST APIs of Elasticsearch and Kibana with Console", but this sounds awkward. 

In my docs I've always said "...in the OpenSearch developer console", but maybe that's not right either. 

Nate, do you have any input?

Reply:
@kolchfa-aws informed me that you guys have already discussed this. LGTM.